bench-tps/src/cli.rs

```diff
@@ -332,10 +338,18 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .help("Controls the connection pool size per remote address; only affects ThinClient (default) \
                     or TpuClient sends"),
         )
+        .arg(
+            Arg::with_name("compute_unit_price")
+            .long("compute-unit-price")
+            .takes_value(true)
+            .validator(|s| is_within_range(s, 0..))
+            .help("Sets constant compute-unit-price to transfer transactions"),
```

Comment:
Can you elaborate the use case for fixed prio fee in benching? With randomization, it triggers "ordering" of transactions by prio fee. From that sense, fixed prio fee is same as no prio fee, is it?

Reply:
A few thoughts here:

1. providing constant priority is slightly different than providing no priority, as we still have overhead processing those additional ixs
2. the use-case I can think of (and have wanted for some time) is for multi-client tests, where clients are sending different sets of transactions w/ differing amounts of conflicts and different priority. i.e. it becomes easy to run 2 clients where 1 sends high-priority transactions with a lot of conflicts, and another sends low-priority non-conflicting transactions.
3. sending to actual networks (testnet, devnet, mnb(?)) - more predictable cost, and generally don't need the randomized values...you just want higher than min priority for that network

Reply:
Ah, thanks for explaining, it makes good sense.